Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/install_git_lfs.py

```diff
@@ -121,6 +124,13 @@ def parser(cls, parser, loggers=None):
             dest='configure',
             default=None,
         )
+        parser.add_argument(
```

Comment:
I'm a bit unclear how a parser helps us here. `mirror_resolver_func` needs to be callable, but you can only define a string via the command line, right? So how does one use the `--mirror-resolver-func`

Reply:
done.